js-html-games/numScript.js

```diff
@@ -0,0 +1,133 @@
+const custom = document.getElementById("custom");
+const random = document.getElementById("random");
+const numForm = document.getElementById("numForm");
+const range = document.getElementById("range");
+const statement = document.getElementById("statement");
+const bttn = document.getElementById("set");
+const hint = document.getElementById("hint");
+let bounds = [];
+let correctNum = 0;
+
+function clear() {
+    bgColor("white");
+    setText("");
+}
+
+function bgColor(color) {
+    document.body.style.backgroundColor = color;
+}
+
+function setText(text) {
+    hint.innerText = text;
+}
+
+function addFormSubmit(corr) {
+    numForm.addEventListener('submit', function (event) {
+        event.preventDefault();
+        const guess = document.getElementById("guess").value;
+        checkNum(guess, corr);
+    })
+}
+
+function hotOrCold(num) {
+    if (num === 0) {
+        bgColor("#8dcc9e");
+        setText("CORRECT! GOOD JOB!");
+    }
+    else if (num > 0 && num <= 10) {
+        bgColor("#e8f781");
+        setText("Close! Keep going! (within 10)");
+    }
+    else if (num > 10 && num <= 100) {
+        bgColor("#f0ce7a");
+        setText("Not quite! (within 100)");
```

Comment:
really like how you used "set" as a prefix for your setText function. I recommend you do the same for `bgColor` (so, do `setBgColor`). Just a general naming convention that most programming languages / clean code use